jerry-core/ecma/operations/ecma-bigint.c

```diff
@@ -120,7 +121,7 @@ ecma_bigint_parse_string (const lit_utf8_byte_t *string_p, /**< string represena
     return ECMA_BIGINT_ZERO;
   }
 
-  while (string_p < string_end_p && *string_p == LIT_CHAR_0)
+  while (string_p < string_end_p && (*string_p == LIT_CHAR_0 || (*string_p == LIT_CHAR_UNDERSCORE && allow_underscore)))
```

Comment:
Could give me an example when the `(*string_p == LIT_CHAR_UNDERSCORE && allow_underscore)` part is needed?

Reply:
without the new expression this input will cause a crash : 0x0_0n